docker/build-aws.sh

```diff
@@ -100,11 +105,14 @@ while true; do
     for (( i=0; i < ${#BUILD_PROJECTS[@]}; i++ ));
     do
         get_build_status ${BUILD_IDS[$i]}
+        if [[ ${DOWNLOAD_SOURCE_STATUS} == "FAILED" ]]; then
```

Comment:
Should we retry if build timeout as well?

Reply:
Done